cterasdk/core/query.py

```diff
@@ -104,7 +112,7 @@ def after(self, value):
         return self.setValue(value)
 
     def setValue(self, value):
-        self.filter._classname = FilterType.fromValue(value)  # pylint: disable=protected-access,attribute-defined-outside-init
+        self.filter._classname = FilterType.fromValue(value, self.reference)  # pylint: disable=protected-access,W0201
```

Comment:
Why do we need the change in the pylint disable list?

Reply:
The line was longer than 150 chars :(